### ISSUE TITLE
Example in REAMDE does not work

### ISSUE THREAD

**Issue by Chris00** [projectatomic/bubblewrap#298]
Running the example in the README yields:
```
$ bwrap --ro-bind /usr /usr --symlink usr/lib64 /lib64 --proc /proc --dev /dev --unshare-pid bash
bwrap: execvp bash: No such file or directory
```

**Comment by Chris00** [comment]
Here is a version that works:
```
bwrap --ro-bind /bin /bin --ro-bind /lib /lib --ro-bind /lib64 /lib64 --unshare-pid bash
```

**Comment by Chris00** [comment]
Since I guess the proper arguments may slightly depend on the system (Debian testing in my case), some additional explanation would be welcome — also because the output in case of failure is not self explanatory...

**Comment by sunaku** [comment]
Agreed. :+1: I think the [simplest and most portable example](https://wiki.archlinux.org/index.php/Bubblewrap#Usage_examples) I have seen comes from [this ArchLinux wiki](https://wiki.archlinux.org/index.php/Bubblewrap):

```sh
$ bwrap --dev-bind / / bash
```

Even if simplistic, having a working "hello world" example inspires confidence in new users. :baby:  In my experience, I certainly remained skeptical about bwrap when the README example didn't _Just Work_. :thinking: It was only later, when I found the minimal example shown above, that I gained confidence :sparkles: in bwrap.

**Comment by mabra** [comment]
If you are not "at home" in linux, this type of "documentation" ist just a big PITA.
host1)
$ bwrap --ro-bind /bin /bin --ro-bind /lib /lib --ro-bind /lib64 /lib64 --unshare-pid bash
bash-5.2#
ls /
ls: error while loading shared libraries: libpcre2-8.so.0: cannot open shared object file: No such file or directory
host2)
$ bwrap --ro-bind /bin /bin --ro-bind /lib /lib --ro-bind /lib64 /lib64 --unshare-pid bash
ERROR: ld.so: object '/usr/lib/x86_64-linux-gnu/libgtk3-nocsd.so.0' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
The whole so, just remains very suspect to me, sorry.
Which of the about 800 distros does this really work???
BTW, nothing of this happens with firejail.
